packages/ui/src/components/pagination/AcvPagination.vue

```diff
@@ -0,0 +1,131 @@
+<script lang="ts" setup>
+  import { computed } from 'vue';
+  import AcvPaginationItem from './AcvPaginationItem.vue';
+  import type { AcvPaginationEvents, AcvPaginationProps } from '@/components';
+
+  const props = defineProps<AcvPaginationProps>();
+  defineEmits<AcvPaginationEvents>();
+
+  const maximumVisibleSlotsInPagination = 7;
+  const lastPage = computed(() => Math.ceil(props.total / props.limit));
+</script>
+
+<template>
+  <div class="acv-pagination">
+    <!-- Previous page button -->
+    <AcvPaginationItem
+      :page="props.modelValue - 1"
```

Comment:
You don't have to write props, you can just write :page=“modelValue - 1”, in my opinion the code will look cleaner that way.

Reply:
yep, my bad